src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -128,6 +128,9 @@ describe('<CourseRunEnrollment />', () => {
 
     const button = screen.getByRole('button', { name: 'Enroll now' });
 
+    // eslint-disable-next-line no-console
+    console.error = jest.fn();
```

Comment:
```suggestion
    spyConsole = jest.spyOn(console, 'error');
```

In this cas `spyOn` is more appropriated IMO.